dvc/command/tag.py

```diff
@@ -50,13 +52,21 @@ def run(self):
                     recursive=self.args.recursive,
                 )
                 if tags:
-                    logger.info(yaml.dump(tags, default_flow_style=False))
+                    logger.info(to_yaml_string(tags))
             except DvcException:
                 logger.exception("failed list tags")
                 return 1
         return 0
 
 
+def to_yaml_string(data, **kw):
```

Comment:
looks like a pretty generic function. I'm surprised that `ruamel` does not provide it. Anyways, let's move to some dvc/utils/yaml or something like that.

Reply:
ruamel doesn't provide this because its author thinks that is inefficient. And I usually don't move utility functions far from there use when they are used only once.

Reply:
It usually leads in a few implementations of the same function eventually. Let's make it private at least so that we can catch it's being used anywhere else.

Reply:
how about `apply_diff`, btw ;) (not saying you should move `apply_diff` to `stage.py`, please keep it in some utils file, so that we can unit test it)

Reply:
There is no way to add `apply_diff()` near its use, only inline :) And what is farther at the end of the file or other file is a questions of metrics you have in mind ;)